client/src/components/NetWorth.tsx

```diff
@@ -64,10 +90,9 @@ export default function NetWorth({
       <h4 className="tableSubHeading">
         A summary of your assets and liabilities
       </h4>
-      <div className="netWorthText">{`Your total Across ${numOfItems} ${pluralize(
-        'Account',
-        numOfItems
-      )}`}</div>
+      <div className="netWorthText">{`Your total Across ${
```

Comment:
Lowercase the "A" in "Across"?

Reply:
Actually, disregard this comment @lindawoo-plaid, as it looks like all of the letters are capitalized anyway (which I noticed when I run the app)